dev_scripts_helpers/thin_client/thin_client_utils.sh

```diff
@@ -31,7 +31,7 @@ dassert_is_sourced() {
     if [[ ! is_sourced ]]; then
         # We are in a script.
         echo -e "${ERROR}: This needs to be sourced and not executed"
-        exit -1
+        die
```

Comment:
Ok. `die` is a bit ominous, I would have called `abort` or `exit_script`, but it's ok with me.